src/components/header/menu/dropdowns/MenuDropdownRun.vue

```diff
@@ -20,6 +20,14 @@
       :shortkey="['ctrl', 'alt', 't']"
       @action="stopSimulation"
     />
+    <MenuDropdownSeparator
+      v-if="simulators && simulators.length"
+    />
+    <MenuDropdownItem
```

Comment:
Are each simulators separate menuItems?

Reply:
Yes. Maybe this should rather be called `formalisms` as a simulator can provide different formalisms?

Reply:
Yes, that sounds better, renew is also letting you choose your formalism that you want to simulate.

Reply:
It's just semantics in the end. The way it is now, there is one simulator for one formalism and a gateway plugin can provide multiple simulators (that may or may not use the same backend).